File: 3617-find-the-original-typed-string-i/find-the-original-typed-string-i.py
class Solution:
    def possibleStringCount(self, word: str) -> int:
        l = 0
        r = 1
        count = 1
        while r <len(word):
            if word[l] == word[r]:
                count+=1
            l+=1
            r+=1
        return count



        

File: 3617-find-the-original-typed-string-i/find-the-original-typed-string-i.java
class Solution {
    public int possibleStringCount(String word) {
        int l =0;
        int r = 1;
        int count = 1;

        while( r < word.length()){
            if(word.charAt(l) == word.charAt(r)){
                count+=1;
            }
            l+=1;
            r+=1;
        }
        return count;
        
    }
}